index.html

```diff
@@ -2,45 +2,199 @@
 
 <head>
     <link rel="stylesheet" href="css/landing.css" />
```

Comment:
landing.css file is no longer needed here so this line can be removed. All of css is now included inside the `<style>` tag and imported from `material-components-web.min.css`